develop/api/NotifyTypes/NT_SNMP_GET.md

```diff
@@ -148,24 +137,201 @@ if (response.Length == 1)
+int elementId = protocol.ElementID;
+string ipPort = "127.0.0.1";
+int multipleGet = 1;
+string instance = string.Empty;
+int connectionId = 0;
+string getCommunityString = null;
+bool splitErrors = false;
+int agentId = protocol.DataMinerID; // For DELT compatibility.
+
+object[] elementInfo = new object[] { elementId, ipPort, multipleGet, instance, connectionId, getCommunityString, splitErrors, agentId };
+
+string sysContactOid = "1.3.6.1.2.1.1.4.0";
+string sysNameOid = "1.3.6.1.2.1.1.5.0";
+string sysLocationOid = "1.3.6.1.2.1.1.6.0:SINGLE";
+
+string[] oidInfo = new[] { sysContactOid, sysNameOid, sysLocationOid };
+
+object[] result = (object[])protocol.NotifyProtocol(295/*NT_SNMP_GET*/, elementInfo, oidInfo);
+
+string sysContactOidValue = Convert.ToString(result[0]);
+string sysNameOidValue = Convert.ToString(result[1]);
+string sysLocationOidValue = Convert.ToString(result[2]);
+```
+
+### TABLE
+
+(Deprecated, use tablev2 instead.)
+
+To retrieve a table via the TABLE method, the entry in oidInfo should be a semicolon-separated list of OIDs where the first entry specifies the OID of the table and the subsequent OIDs specify the OIDs of the columns that should be retrieved (e.g. `1.3.6.1.2.1.2.2.1;1.3.6.1.2.1.2.2.1.2;1.3.6.1.2.1.2.2.1.3:TABLE`).
+
+This will result in GetNextRequest PDUs being sent out (to retrieve the instance values) followed by multiple GetRequest PDUs containing multiple variable bindings (to retrieve the column values). The result object is a table that has as its first column the instance values. The remaining columns are the values of the columns specified in the request.
+
+```csharp
+int elementId = protocol.ElementID;
+string ipPort = "127.0.0.1";
+int multipleGet = 0;
+string instance = string.Empty;
+int connectionId = 0;
+string getCommunityString = null;
+bool splitErrors = false;
+int agentId = protocol.DataMinerID; // For DELT compatibility.
+
+object[] elementInfo = new object[] { elementId, ipPort, multipleGet, instance, connectionId, getCommunityString, splitErrors, agentId };
+
+string ifEntry = "1.3.6.1.2.1.2.2.1";
+string ifDescr = "1.3.6.1.2.1.2.2.1.2";
+string ifType = "1.3.6.1.2.1.2.2.1.3";
+
+string[] oidInfo = new string[] { $"{ifEntry};{ifDescr};{ifType}:TABLE" };
+
+var result = (object[])protocol.NotifyProtocol(295/*NT_SNMP_GET*/, elementInfo, oidInfo);
+
+object[] table = (object[])result[0];
+
+for(int i=0; i<table.Length; i++)
+{
+  var column = (object[])table[i];
+
+  for(int j=0; j<column.Length; j++)
+  {
+    var cellValue = Convert.ToString(column[j]);
+    //// ...
+  }
+}
+```
+
+### TABLEV2
+
+To retrieve a table via the TABLEV2 method, the entry in oidInfo should be a semicolon-separated list of OIDs of the columns that should be retrieved (e.g. `1.3.6.1.2.1.2.2.1.2;1.3.6.1.2.1.2.2.1.3:TABLEV2`).
+
+This will result in GetBulkRequest PDUs being sent out. The number of maxRepitions to use in the GetBulkRequest can be specified in the elementInfo object. The default value is 6.
```

Comment:
Could it be that this should be "maxRepetitions" instead of "maxRepitions"?

Reply:
@MariekeGO: indeed, that is a typo.